spec/features/ping_spec.rb

```diff
@@ -1,5 +1,11 @@
 require 'rails_helper'
+
```

Comment:
should bypass authentication for ping, because healthcheck should be able to use it

Reply:
Ah yes, forgot about that. Will update it on this branch